incubator/spark-history-server/charts/nfs/templates/nfs-pvc.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if .Values.enableExampleNFS }}
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: {{ .Values.pvcName }}
+  namespace: {{ .Release.Namespace }}
```

Comment:
Please remove namespace everywhere. It is unnecessary and handled by Helm.

Reply:
Thanks for the reminder! I removed all namespaces except for the one shown below in `rbac.yaml`:

```
apiVersion: rbac.authorization.k8s.io/v1beta1
kind: ClusterRoleBinding
metadata:
  name: {{ .Values.clusterRoleBindingName }}
subjects:
- kind: ServiceAccount
  name: spark-history-server
  namespace: {{ .Release.Namespace }}
roleRef:
  kind: ClusterRole
  name: {{ .Values.clusterRoleName }}
  apiGroup: rbac.authorization.k8s.io
```

Because the namespace for a `ClusterRoleBinding` is required. Without it, `helm install` would fail even with `--namespace` flag.